src/pkgs-lib/shell/flk.sh

```diff
@@ -74,6 +74,8 @@ case "$1" in
   "iso")
     nix build \
       "$DEVSHELL_ROOT#nixosConfigurations.$2.config.system.build.iso" \
+      -o \
+      "$DEVSHELL_ROOT/iso/$2" \
```

Comment:
```suggestion
      "$DEVSHELL_ROOT/iso/$2.iso" \
```
Would that make sense? or is it redundant since the file is already in a iso folder?

Reply:
Actually the file is just a symlink and it is right now just `result` but I agree, it is probably clearer.
What do you think it should be for a symlink, is there a standard for that?

Reply:
I just tested `nix build -o`(didn't know that was a thing), I would think `flk iso NixOS` would output as `iso/NixOS`. Is that right? Because if thats true then I think the `.iso` is unnecessary. If it outputs as `iso/result` then thats a little weird.

Reply:
yes, it ouputs as `iso/NixOS` instead of `./result` which is way better.
If you want to, I can change it to `iso/NixOS.iso`, it's a matter of standard more than anything, should a link to a file respect the extension of the pointed file?

Reply:
Ok perfect.
> If you want to, I can change it to iso/NixOS.iso, it's a matter of standard more than anything, should a link to a file respect the extension of the pointed file?

Good point, I think theres also a argument for clarity that having the `.iso` extension is more obvious. And some gui iso flashers might only show .iso files in the file selector.

So yeah I vote for a `.iso` extension for the file, even though its a symlink.

Reply:
> And some gui iso flashers might only show .iso files in the file selector.
Oh! that's a really good point, I'll change it, yes .iso should be clearer as well ^^